src/coreclr/jit/compiler.h

```diff
@@ -6862,7 +6862,7 @@ class Compiler
     bool optExtractInitTestIncr(
         BasicBlock** pInitBlock, BasicBlock* bottom, BasicBlock* top, GenTree** ppInit, GenTree** ppTest, GenTree** ppIncr);
 
-    void optRedirectBlock(BasicBlock*      blk,
+    void optInitDuplicatedBlockTargets(BasicBlock*      blk,
```

Comment:
Not sold on the name -- `optSetMappedBlockTargets`?

But we can sort it out later.

Reply:
@jakobbotsch how do you feel about `optSetMappedBlockTargets`?

Reply:
Thanks, I'll fix it now then